docs/articles/joss/paper.md

```diff
@@ -81,9 +81,9 @@ Table 1: GeoClimate features description and differences with similar commonly-u
 
 GeoClimate performs indicators computation at three spatial unit scales, a spatial unit being a POLYGON or MULTIPOLYGON geometry:
 
-1. Building scale, defined as a collection of features represented by 2.5 geometries with attributes as measures for walls and roof, 
-2. Block scale, defined as a set of buildings touching each other (at least one point in common) or as an isolated building, 
-3. Reference Spatial Unit (RSU) scale, being the elementary unit to characterize all the characteristics of a piece of land (not only related to buildings but also to vegetation, water). 
+1. Building scale, defined as a collection of features represented by 2D geometries with attributes as measures for walls and roof. 
```

Comment:
keep between parenthesis 2.5D at the end of this sentence ?

Reply:
where ?